limit-break/objdef.cpp

```diff
@@ -0,0 +1,27 @@
+#include "stdafx.h"
+#include "objdef.h"
+
+void Ring_Main_SkyDeck(ObjectMaster *a1) {
+	if (a1->Data1->Scale.x == -1)
+		a1->Data1->Scale.x = 4;
+
+	Ring_Main(a1);
+}
+
+float __stdcall squareroot_r(float a1) {
+	return 0;
+}
+
+void objdef_init() {
+	float range = 3600000;
+
+	//Bridge in EC01
+	WriteData((float*)0x7EC9F8, range);
```

Comment:
Is this the distance at which the bridge appears? If so, this ought to also use `clip_current` from `clip.h`

Reply:
Oh yeah it should use clip_current, in that case it should replace the pointer in the bridge function directly.